test/lib/BedrockTester.cpp

```diff
@@ -42,164 +27,113 @@ string BedrockTester::getTempFileName(string prefix) {
     return buffer;
 }
 
-BedrockTester::BedrockTester(const string& filename, const string& serverAddress, const list<string>& queries, const map<string, string>& args, bool wait) {
-    nextActivity = 0;
-    if (filename.empty()) {
-        _dbFile = BedrockTester::DB_FILE;
-    } else {
-        _dbFile = filename;
-    }
-
-    if (serverAddress.empty()) {
-        _serverAddr = BedrockTester::SERVER_ADDR;
-    } else {
-        _serverAddr = serverAddress;
-    }
-
-    createFile(_dbFile);
-
-    // We don't use SQLite here, because we specifically want to avoid dealing with journal tables.
-    if (queries.size()) {
-        sqlite3* _db;
-        sqlite3_initialize();
-        sqlite3_open_v2(_dbFile.c_str(), &_db, DB_WRITE_OPEN_FLAGS, NULL);
-
-        for (string query : queries) {
-            int error = sqlite3_exec(_db, query.c_str(), 0, 0, 0);
-            if (error) {
-                cout << "Init Query: " << query << ", FAILED. Error: " << error << endl;
-            }
+string BedrockTester::getServerName() {
+    for (auto location : locations) {
+        if (SFileExists(location)) {
+            return location;
         }
-
-        SASSERT(!sqlite3_close(_db));
     }
-
-    if (startServers) {
-        startServer(args, wait);
-    }
-}
-
-BedrockTester::~BedrockTester() {
-    if (db) {
-        delete db;
-        db = 0;
-    }
-    if (writableDB) {
-        delete writableDB;
-        writableDB = 0;
-    }
-    if (serverPID) {
-        stopServer();
-        deleteFile(_dbFile);
-    }
-}
-
-SQLite& BedrockTester::getSQLiteDB() {
-    if (!db) {
-        db = new SQLite(_dbFile, 1000000, 100, 3000000, -1, -1);
-    }
-    return *db;
+    return "";
 }
 
-SQLite& BedrockTester::getWritableSQLiteDB() {
-    if (!writableDB) {
-        writableDB = new SQLite(_dbFile, 1000000, 100, 3000000, -1, -1);
+void BedrockTester::stopAll() {
+    for (auto p : _testers) {
+        p->stopServer();
     }
-    return *writableDB;
 }
 
-// This is sort of convoluted because of the way it was originally built. We can probably just have this always
-// use the member variable db and never open it's own handle. This was added early ob for debugging and should be
-// obsolete.
-string BedrockTester::readDB(const string& query) { return getSQLiteDB().read(query); }
-
-bool BedrockTester::readDB(const string& query, SQResult& result) { return getSQLiteDB().read(query, result); }
```

Comment:
Why are you removing this?

Reply:
It's unused (it was even marked obsolete in the comment that's removed). There's an equivalent function added in AuthTester where it is used.

Reply:
Ok, added back.